requirements.txt

```diff
@@ -4,6 +4,5 @@ endesive
 lxml
```

Comment:
This and other dependencies are also Odoo base ones, e.g. `xlrd`. I'd remove all non-external dependencies in the same commit